File: ScpProject/StrengthConApp/src/app/Models/MultiMedia/Movie.ts
import { ITaggable } from './../../Interfaces/ITaggable';
import { Tag } from './../Tag';

export class Movie implements ITaggable {
    Id: number;
    Name: string;
    IsDeleted: boolean;
    CanModify: boolean;
    Url: string;
    CreatedUserId: number;
    WeekIds :number[];
    Tags: Tag[] = [];
}

File: ScpProject/BL/BusinessObjects/MultiMedia/Movie.cs
using System.Collections.Generic;

namespace BL.BusinessObjects.MultiMedia
{
    public class Movie
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string URL { get; set; }
        public List<Tag> Tags { get; set; }
        public bool CanModify { get; set; }
        public bool IsDeleted { get; set; }
    }
}
